controller/client/client.go

```diff
@@ -813,6 +813,14 @@ func (c *Client) Backup() (io.ReadCloser, error) {
 	return res.Body, err
 }
 
+// GetBackupMeta returns metadata for latest backup
+func (c *Client) GetBackupMeta() (*ct.ClusterBackup, error) {
+	h := http.Header{"Accept": []string{"application/json"}}
+	b := &ct.ClusterBackup{}
+	_, err := c.RawReq("GET", "/backup", h, nil, b)
```

Comment:
Looks like this doesn't break anything :) (https://ci.flynn.io/builds/20160409135114-7ba1d9c3)